src/api/v3/single_page.py

```diff
@@ -0,0 +1,41 @@
+"""
+View to return a JSON representation of a single page. The page can
+be selected via the id or the permalink.
+"""
+from django.http import JsonResponse, HttpResponse
```

Comment:
```suggestion
from django.http import JsonResponse, Http404
```
This is needed if you apply my suggestion below.